networking/network_observability/installing-operators.adoc

```diff
@@ -1,13 +1,27 @@
 :_content-type: ASSEMBLY
 [id="installing-network-observability-operators"]
-= Installing Network Observability operators in {product-title}
+= Installing the Network Observability operator
 include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
+Installing Loki is a prerequisite for using the Network Observability operator. It is recommended to install Loki using the Loki Operator; therefore these steps are documented below prior to the Network Observability Operator installation.
 
-The External DNS Operator deploys and manages `ExternalDNS` to provide the name resolution for services and routes from the external DNS provider to {product-title}.
+The Loki Operator integrates a gateway that implements multi-tenancy & authentication with Loki for data flow storage. The *LokiStack* resource manages *Loki*, which is a scalable, highly-available, multitenant log aggregation system, and a web proxy with {product-title} authentication. The *LokiStack* proxy uses {product-title} authentication to enforce multi-tenancy and facilitate the saving and indexing of data in *Loki* log stores. 
 
-include::modules/nw-external-dns-operator.adoc[leveloffset=+1]
+[NOTE]
+====
+The Loki Operator can also be used for xref:../../logging/cluster-logging-loki.adoc#cluster-logging-loki[Logging with the LokiStack]. The Network Observability operator requires a dedicated LokiStack separate from Logging.
+====
+
+include::modules/network-observability-loki-install.adoc[leveloffset=+1]
+include::modules/network-observability-lokistack-create.adoc[leveloffset=+2]
+include::modules/network-observability-roles-create.adoc[leveloffset=+1]
+include::modules/network-observability-kafka-option.adoc[leveloffset=+1]
+include::modules/network-observability-operator-install.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Additional resources should have [role="_additional-resources"] on the preceding line.